### PR TITLE
HEEDLS-298 Add tutorial customisation back end

### PR DESCRIPTION
**Pull request by benjimarshall**
### Changes
* Add a "Supporting information" field to the course settings object
* Get course settings when getting tutorial information

### Testing
Add some more unit tests to the course settings and tutorial service,
tweak some existing ones to make constructor signatures match.

### CODE REVIEW

File: DigitalLearningSolutions.Data.Tests/Models/CourseSettingsTests.cs [hunk 1]
```diff
@@ -190,5 +190,34 @@ public void CourseSettings_should_have_default_ConsolidationExercise_when_wrong_
...
+        {
+            // When
+            var courseSettings = new CourseSettings("{\"lm:si\":\"Supporting information\"}");
+
+            // Then
+            courseSettings.SupportingInformation.Should().Be("Supporting information");
+        }
+
+        [Test]
+        public void CourseSettings_should_have_default_SupportingInformation_when_not_in_json()
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data.Tests/Models/CourseSettingsTests.cs:204]: One thing I hadn't considered before, for string values in the course setting we should probably not use them if they're empty 🙂

**Review by FridaTveit — COMMENTED** [review]
One suggestion, otherwise looks good 🙂

File: DigitalLearningSolutions.Data.Tests/Models/CourseSettingsTests.cs (hunk 1 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Data.Tests/Models/CourseSettingsTests.cs:225]: Yeah neither had I! Fixed in 72ce2a9 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**